crates/sqlite-lambda/tests/snapshots/examples__type_conversions.snap

```diff
@@ -0,0 +1,76 @@
+---
+source: crates/sqlite-lambda/tests/examples.rs
+expression: output
+---
+[
+  [
+    {
+      "case": "true",
+      "output": 1
```

Comment:
Does this mean that a sqlite lambda that selects fields out of a document with boolean values will have outputs `0` and `1`? I guess that's how sqlite stores booleans but it probably wouldn't be what a user would expect.

Reply:
That's correct. It's unavoidable, because SQLite doesn't meaningfully distinguish booleans and text in it's returned value types.

Users _can_ work around this using sqlite's json functions and the `flow_document` output special case:
`select json_object('bool', json('true'), 'answer', 42) as flow_document from ... `. SQLite has some internal magic to thread through nested values of json functions that aren't exposed in the public API.